src/lib/buySellHelper/buySellHelper.ts

```diff
@@ -0,0 +1,84 @@
+import { TradesDataType, TRADE_STATUS } from "@jaws/db/tradesMeta";
+import { isToday } from "@jaws/lib/helpers";
+import { BuySellConstants, DEFAULT_BUY_SELL_CONFIG } from "./buySellConstants";
+
+export const getBuySellHelpers = (config?: Partial<BuySellConstants>) => {
+  const _config = { ...DEFAULT_BUY_SELL_CONFIG, ...config };
+
+  return {
+    get config() {
+      return { ..._config } as const;
+    },
+
+    /**
+     * Returns the maximum amount of money (total value) the position is
+     * allowed to drop before triggering Stop loss (1)
+     */
+    getStopLossMaxAmount: (totalAssets: number): number =>
+      totalAssets * _config.STOP_LOSS_1_PORTFOLIO_PERCENTAGE,
```

Comment:
Just an opinion from my perpective: It seems like these constants are only used here. I think I would prefer to have them in the same file so that it is easier to see what they are.

Reply:
I'll move the default values to this file and keep the type definition separate